akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -356,30 +398,30 @@ object DirectoryListing {
   def directoryMarshaller(renderVanityFooter: Boolean): ToEntityMarshaller[DirectoryListing] =
     Marshaller.StringMarshaller.wrapWithEC(MediaTypes.`text/html`) { implicit ec ⇒ listing ⇒
       val DirectoryListing(path, isRoot, files) = listing
```

Comment:
Yeah, I guess so. Prevented me from getting rid of `java.io.File` completely, though.